source/grid/tria.cc

```diff
@@ -10605,6 +10605,59 @@ Triangulation<dim,spacedim>::end_face () const
 }
 
 
+/*------------------------ Vertex iterator functions ------------------------*/
+
+
+template <int dim, int spacedim>
+typename Triangulation<dim,spacedim>::vertex_iterator
+Triangulation<dim,spacedim>::begin_vertex() const
+{
+  if (dim==1)
+    {
+      Assert(false, ExcNotImplemented());
+      return raw_vertex_iterator();
```

Comment:
I see. Then document this.

It's a bit sad that we can't make this work in a dimension independent fashion, but I don't think that there will be a lot of uses for this functionality anyway -- we always recommend to work from cells to faces to edges, and so there are not many places where one would want to iterate over the vertices.